target/linux/lantiq/patches-5.15/0008-MIPS-lantiq-backport-old-timer-code.patch

```diff
@@ -1005,15 +1007,24 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
 +	}
 +
 +	for (i = 0; i < timer_dev.number_of_timers; i++) {
-+		ret = request_irq(TIMER_INTERRUPT + i, timer_irq_handler, IRQF_TIMER, gptu_miscdev.name, &timer_dev.timer[i]);
```

Comment:
If the `TIMER_INTERRUPT` define is not used any more, you can remove it too.

Reply:
The `TIMER_INTERRUPT` is still used in the `timer_irq_handler` to get calculate the timer index out of the irq number at the moment.